tools/docker/builder/openwrt/Dockerfile

```diff
@@ -81,8 +81,9 @@ RUN echo "src-git prpl ${PRPL_FEED}" >> feeds.conf.default \
     && scripts/feeds install -a \
     && cp "/home/openwrt/prplmesh_config" .config \
     && make defconfig \
-    && make -j$(nproc) V=sc \
-    && make -j$(nproc) \
+    && cat .config \
```

Comment:
Although it's being dumped by the defconfig file, it won't be visible directly in the CI log, so I think it's still ok.
It will only be visible in the build.log file, which already contains a very verbose build output.

It's currently not possible to add the file as an artifact, because the CI does not run _inside_ the container (if we really want to have it as an artifact, we could still copy it out of the container).